.github/workflows/security-secrets.yml

```
@@ -20,6 +20,7 @@ jobs:
   scanning:
     name: GitGuardian scan
     runs-on: ubuntu-latest
+    if: secrets.GITGUARDIAN_API_KEY != ''
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)